src/krkn_lib_kubernetes/client.py

```diff
@@ -85,11 +85,7 @@ def __initialize_clients(self, kubeconfig_path: str = None):
             kubeconfig_path = os.path.expanduser(kubeconfig_path)
 
         try:
-            f = open(kubeconfig_path)
-            with f:
-                kubeconfig_str = f.read()
-                self.__initialize_clients_from_kconfig_string(kubeconfig_str)
-
+            config.load_kube_config(kubeconfig_path)
```

Comment:
nevermind wont work

Reply:
```
            config.load_kube_config(kubeconfig_path)
            self.api_client = client.ApiClient()
            self.k8s_client = config.new_client_from_config(config_file=kubeconfig_path)
            self.cli = client.CoreV1Api(self.k8s_client)
            self.batch_cli = client.BatchV1Api(self.k8s_client)
            self.custom_object_client = client.CustomObjectsApi(self.k8s_client)
            self.dyn_client = DynamicClient(self.k8s_client)
            self.watch_resource = watch.Watch()
```

Reply:
think if we put the above in the __initialize_clients_from_kconfig_string instead of calls to arcaflow functions I think we'll be good
And take out 

```
    def __del__(self):
        self.api_client.rest_client.pool_manager.clear()
        self.api_client.close()
```

Reply:
Added @paigerube14. Thanks!